src/tensor/copy_intersection.cc

```diff
@@ -434,6 +434,11 @@ void copy_intersection_async<int64_t>(const Tensor<int64_t> &src,
         const std::vector<Index> &src_offset, const Tensor<int64_t> &dst,
         const std::vector<Index> &dst_offset);
 
+template
+void copy_intersection_async<bool_t>(const Tensor<bool_t> &src,
+        const std::vector<Index> &src_offset, const Tensor<bool_t> &dst,
+        const std::vector<Index> &dst_offset);
+
```

Comment:
copy_intersection<bool_t> is missing (without _async suffix)

Reply:
Probably error in github. It is actually in this file, little lower

Reply:
done